go.mod

```diff
@@ -1,172 +1,168 @@
 module github.com/sighupio/furyctl
 
-go 1.22
-
-toolchain go1.22.3
+go 1.23.2
```

Comment:
do we want to pin the patch version too? Or would it be better to use `1.23` instead of `1.23.2` like we did for `1.22`?

Reply:
`go` with patch version is the replacement of the old `toolchain`